Automattic/wp-calypso
# Package version of What's New Guide component

This PR is a refactor of the What's New component discussed in #49957.

The two big changes are:
* Moved to /packages to make the component available in both Calypso and Editing Toolkit.
* Uses `wpcom.req()` for the api fetch rather than Redux

I've also wrapped the data in the Guide component since that's how it will be used.

#### Testing instructions
* Sandbox `public-api`
* Patch your WPcom sandbox with D56831-code 
* Patch your MC sandbox with D57620-code
* Navigate to `/mobile-announce/whats-new.php` on your MC sandbox
* Add some new announcements to the tool
* Checkout this PR and navigate to `/marketing/tools/SITESLUG`
* Confirm that the What's New Guide component appears and works as expected
* Switch your language preference and confirm that API sends the correct strings
* Delete the What's New announcements that you added.

Note: It's on the marketing page right now for the sake of testing; that's not the final implementation.

Related to #
#49957 
#48722 
D56831-code
D57620-code
